routing.rst

```diff
@@ -1203,7 +1203,7 @@ the common configuration using options when importing the routes.
             # Uncomment this option to make that URL "/blog" instead
             # trailing_slash_on_root: false
             # you can optionally exclude some files/subdirectories when loading annotations
-            # exclude: '../../src/Controller/{DebugEmailController}.php'
+            exclude: '../../src/Controller/{DebugEmailController}.php'
```

Comment:
This change should be reverted. We want to display this option commented. In XML is not commented because it's not possible to do that because you cannot add a comment inside a tag. But that's just an XML problem and shouldn't affect others. Thanks!

Reply:
4 lines of "real" comments, and then suddenly 1 line of commented-out code? This is hard to read. Besides, I can't see any reason why `exclude` should be "downgraded" - it's just as legitimate as `requirements`